src/models/sheltersModel.js

```diff
@@ -13,6 +14,27 @@ const getAllShelters = async (params) => {
     });
 }
 
+const createShelters = async (params) => {
+
+    var vals = `${SqlUtil.SQ(params.shelterName)}, ${SqlUtil.SQ(params.address)}, ${SqlUtil.SQ(params.emailAddress)},
```

Comment:
This format is different than the format in Esther's PR. I think we should have a discussion on Teams as to which we want to use

Reply:
fixed! thanks!